firmware/tunerstudio/rusefi.input

```diff
@@ -123,7 +123,7 @@ enable2ndByteCanID = false
 
 		filter = VBatt, "VBatt", VBatt       <       , 12,      , true
 
-;         filter = minTps, "Minimum TPS", TPS,       >       , 1,       , true
+         filter = minTps, "Minimum TPS", TPSValue,       <       , 1,       , true
```

Comment:
This allows to use Online Auto-tune correctly with Coasting Fuel Cut-off enabled!

Reply:
This fix is tested on a real car by @darxfame